### PR TITLE
61 individual feedback

### PR DESCRIPTION
**Pull request by noamstul**
Adds the option for individual feedback, currently **only for Multiple Select question types**

SR only -- rereads their responses, tells them what they selected, and tells them if it was correct or incorrect, EN & FR working

### CODE REVIEW

File: public/m1-7_en.txt
Comment (r-torres-csps) [inline public/m1-7_en.txt:1]: good

File: public/m1-7_fr.txt
Comment (r-torres-csps) [inline public/m1-7_fr.txt:1]: good, will update with proper text

File: src/App.vue
Comment (r-torres-csps) [inline src/App.vue:1]: All good to me

File: src/components/Question.vue
Comment (r-torres-csps) [inline src/components/Question.vue:1]: RM instructions, rm aria-live attr, add p tags to sr-feedback. Check #66 to improve code

**Review by r-torres-csps — CHANGES_REQUESTED** [review]
Check #66 for my suggested changes wich are:

`RM instructions, rm aria-live attr, add p tags to sr-feedback`

I already made the changes in that branch.

**Comment by r-torres-csps** [comment]
## Question 2 shows generic feedback by default
I have not selected any options in the second question and it shows the generic feedback without icons. It looks the same in both EN and FR. 
image

**Comment by noamstul** [comment]
> ## Question 2 shows generic feedback by default
> I have not selected any options in the second question and it shows the generic feedback without icons. It looks the same in both EN and FR. image

Resolved

**Review by r-torres-csps — APPROVED** [review]
Looks good

**Pull request merged**